### PR TITLE
ControllerManager Fix For Teleport

### PR DESCRIPTION
**Pull request by StayTalm**
Companion PR For: https://github.com/Unity-Technologies/XR-Interaction-Toolkit-Examples/pull/13

### CODE REVIEW

File: VR/Assets/Scripts/ControllerManager.cs [hunk 1]
```diff
@@ -4,15 +4,28 @@
 using UnityEngine.XR;
 using UnityEngine.XR.Interaction.Toolkit;
 
+[DefaultExecutionOrder(10)]
```
Comment (StayTalm) [inline VR/Assets/Scripts/ControllerManager.cs]: This needs a constant

File: VR/Assets/Prefabs/XRRig_Demo.prefab [hunk 2]
```diff
@@ -1330,7 +1330,8 @@ MonoBehaviour:
   m_RigBaseGameObject: {fileID: 1717954561456925282}
   m_CameraFloorOffsetObject: {fileID: 1717954561314460871}
   m_CameraGameObject: {fileID: 1717954562053134886}
-  m_TrackingSpace: 0
+  m_TrackingOriginMode: 1
```
Comment (StayTalm) [inline VR/Assets/Prefabs/XRRig_Demo.prefab]: This should be reverted

File: VR/Assets/Prefabs/XRRig_Demo.prefab [hunk 3]
```diff
@@ -950,9 +950,9 @@ MonoBehaviour:
   m_UpdateTrackingType: 0
   m_EnableInputActions: 1
   m_ControllerNode: 4
-  m_SelectUsage: 2
-  m_ActivateUsage: 2
-  m_UIPressUsage: 2
+  m_SelectUsage: 5
```
Comment (StayTalm) [inline VR/Assets/Prefabs/XRRig_Demo.prefab]: I need to investigate if these changed as intended.

**Comment by soraryu** [comment]
I think you have tested against an unreleased version of the XR Interaction Toolkit. The samples now break with 0.9.2, since InputHelpers does not exist there. See #15

**Pull request merged**